classes/dl-srcpkg.bbclass

```diff
@@ -0,0 +1,59 @@
+#
+# This class enables getting source code of all packages which are included in rootfs.
+#
+# Copyright (C) Cybertrust Japan Co., Ltd.
+#
+# SPDX-License-Identifier: MIT
+#
+
+#
+# [How to use]
+# Add following line into the local.conf.
+# INHERIT += "dl-srcpkg"
+#
+
+LOCAL_APT_DIR="tmp/isar-apt/${DISTRO}-${DISTRO_ARCH}/apt/${DISTRO}"
```

Comment:
tmp directory is changeable so ${TMPDIR} would be better.

Reply:
TMPDIR is a absolute path from "/" on HOST PC environment. This LOCAL_APT_DIR is relative directory in rootfs environment. So we can't use TMPDIR.